app/javascript/angular/code/services/_fixed_sessions.js

```diff
@@ -91,7 +92,9 @@ export const fixedSessions = (
       if ($window.location.pathname !== constants.fixedMapRoute) return;
 
       this.drawSessionsInLocation();
-      this.fetchableSessionsCount = fetchableSessionsCount;
+      if (fetchableSessionsCount) {
+        this.fetchableSessionsCount = fetchableSessionsCount;
```

Comment:
or `fetchableSessionsCount || 0` if you prefer (I like the shortness but don't have strong preferences)

Reply:
That wouldn't be the same effect? I don't want to set it to 0, I want to preserve the original value that's in `this.fetchableSessionsCount`

Reply:
ah, ok, I just saw the 0 default value above. so I guess it can be changed somewhere in between, ok